src/Application/Service/Property/PropertyViewService.js

```diff
@@ -1,19 +1,17 @@
-var PropertyDoesNotExist = require('../../../Domain/Model/Property/PropertyDoesNotExist')
+import PropertyDoesNotExist from '@Domain/Error/Property/PropertyDoesNotExist';
 
-module.exports = class PropertyViewService {
-    constructor(
-      PropertyRepository
-    ) {
-      this.propertyRepository = PropertyRepository;
-    }
-
-    execute(PropertyViewRequest) {
-        var property = this.propertyRepository.ofId(PropertyViewRequest.propertyId)
-        console.log('eee')
-        if (property === null || property.owner !== PropertyViewRequest.ownerId) {
-            throw new PropertyDoesNotExist()
-        }
+export default class PropertyViewService {
+  constructor(PropertyRepository) {
+    this.propertyRepository = PropertyRepository;
+  }
 
-        return property
+  execute(PropertyViewRequest) {
+    var property = this.propertyRepository.ofId(PropertyViewRequest.propertyId);
+    console.log('eee');
```

Comment:
This was my mistake xD

Reply:
Well, not working after moving to ES6 yet. But tests do ;)

Reply:
You can remove line 10